lib/kbsecret/cli/kbsecret-cp

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "kbsecret"
+require "tty-prompt"
+
+include KBSecret
+
+cmd = CLI.create do |c|
+  c.slop do |o|
+    o.banner = <<~HELP
+      Usage:
+        kbsecret cp [options] <target-session> <record> [record]
```

Comment:
I think the `[record]` at the end of the line here is erroneous -- this command only takes on record to copy, right?

Reply:
I had trouble parsing target-session argument together with a list of records at the end. So I figured I implement it for a single record first. I think this command should work with one or more records, I'll fix it.